my-app/src/Pages/MainPage.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+class MainPage extends Component {
+  init() {
+    // check isAuthorized
+    // if Yes -- redirect to departments
+    // if NO -- redirect to Login
+  }
+
+  componentDidMount() {
+    this.init();
```

Comment:
Why do you need this lifecycle if it call method what does nothing, and why do you need this method?

Reply:
I forgot to delete the Main Page, it's not necessary at al=))